cmake/Modules/AddBoostLib.cmake

```diff
@@ -34,4 +34,6 @@ function(_add_boost_lib)
   if(BOOSTLIB_INCLUDE_PRIVATE)
     target_include_directories(Boost_${BOOSTLIB_NAME} PRIVATE ${BOOSTLIB_INCLUDE_PRIVATE})
   endif()
+
+  message("Added Boost::${BOOSTLIB_NAME}")
```

Comment:
Could you remove this bit? I'm afraid it might be a bit spammy.

Reply:
Sure! Was mostly just to verify the behavior.